src/components/navbar/RightBlock.tsx

```diff
@@ -83,42 +90,64 @@ const AccountContainer = ({
   )
 }
 
-export default function ({
+const SuspendedAccount = ({
   account,
   needNetworkChange,
   eNSName,
-}: {
-  account?: string
-  needNetworkChange: boolean
-  eNSName?: string
-}) {
+}: AccountProps) => {
   return (
-    <div className={walletContainer}>
-      <SocialLinks />
-      <SealVerse />
-      <LastDelimiter />
-
-      <Suspense
-        fallback={
-          <div className={accountLinkContainer}>
-            <div className={walletAccount}>
-              <AccentText small color="text-primary-semi-dimmed">
-                <span className={displayTo('lg')}>Fetching...</span>
-                <span className={displayFrom('lg')}>Fetching account...</span>
-              </AccentText>
-            </div>
-            <div className={width('w-fit')}>
-              <Logo connected={false} />
-            </div>
+    <Suspense
+      fallback={
+        <div className={accountLinkContainer}>
+          <div className={walletAccount}>
+            <AccentText small color="text-primary-semi-dimmed">
+              <span className={displayTo('lg')}>Fetching...</span>
+              <span className={displayFrom('lg')}>Fetching account...</span>
+            </AccentText>
```

Comment:
We should place it into a separate component and call it `NavLoading` or similar. It's hard to read this part